CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ A breaking change will get clearly marked in this log.
 
 ## Unreleased
 
+## [v11.3.0](https://github.com/stellar/js-stellar-sdk/compare/v11.2.2...v11.3.0)
+
+### Added
+* Introduces an entire suite of helpers to assist with interacting with smart contracts ([#929](https://github.com/stellar/js-stellar-sdk/pull/929)):
+  - `ContractClient`: generate a class from the contract specification where each contract method gets included with a "JSified" name. Each method returns an `AssembledTransaction` that can be used to modify, simulate, decode results, and possibly sign, & submit the transaction.
```

Comment:
The name is no longer JSified. Did you copy-paste this from one of my notes? I can send a PR to update the source.

```suggestion
  - `ContractClient`: generate a class from the contract specification, with a method for each original contract method. Each returns an `AssembledTransaction` that can be used to modify, simulate, decode results, and possibly sign, & submit the transaction.
```

Reply:
yeah I ripped this from the js docstrings

Reply:
fixed! thanks :+1: